src/main/java/controller/AttendanceController.java

```diff
@@ -0,0 +1,169 @@
+    }
+
+    private LocalTime readAttendanceTime() {
+        outputView.printAddAttendanceDate();
+        return inputView.readTime();
+    }
+
+    private String inputCrewName() {
+        outputView.printAddAttendanceCrewName();
+        final String crewName = inputView.readCrewName();
+        attendanceBook.validateCrewByName(crewName);
+        return crewName;
+    }
+
+    private LocalDateTime convertLocalDateTime(final LocalTime time) {
+        return LocalDateTime.of(LocalDate.of(2024, 12, LocalDate.now().getDayOfMonth()), time);
+    }
+
+    private void updateAttendance() {
+        final String crewName = LoopTemplate.tryCatchLoop(this::inputCrewNameForUpdate, outputView);
+        final int dayOfMonth = LoopTemplate.tryCatchLoop(this::inputDayOfMonthForUpdate, crewName, outputView);
+        modifyAttendance(crewName, dayOfMonth);
+    }
+
+    private void modifyAttendance(final String crewName, final int dayOfMonth) {
+        final LocalTime targetTime = LoopTemplate.tryCatchLoop(this::inputUpdateTime, outputView);
+        final Attendance beforeAttendance = attendanceBook.updateAttendanceByCrewNameAndDay(targetTime, crewName,
+                dayOfMonth);
+        final LocalDateTime localDateTime = LocalDateTime.of(LocalDate.of(2024, 12, dayOfMonth), targetTime);
+        outputView.printUpdateAttendanceResult(convertAttendanceToResponse(beforeAttendance), targetTime,
+                AttendanceStatus.of(localDateTime));
+    }
+
+    private LocalTime inputUpdateTime() {
+        outputView.printUpdateAttendanceDate();
+        return inputView.readTime();
+    }
+
+    private int inputDayOfMonthForUpdate(final String crewName) {
+        outputView.printUpdateAttendanceDayOfMonth();
+        final int dayOfMonth = inputView.readDayOfMonth();
+        attendanceBook.validateUpdateAttendanceDay(crewName, dayOfMonth);
+        return dayOfMonth;
+    }
+
+    private String inputCrewNameForUpdate() {
+        outputView.printUpdateAttendanceCrewName();
+        final String crewName = inputView.readCrewName();
+        attendanceBook.validateCrewByName(crewName);
+        return crewName;
+    }
+
+
+    private void responseCrewAttendanceHistory() {
+        final String crewName = LoopTemplate.tryCatchLoop(this::inputCrewName, outputView);
+        final Crew crew = attendanceBook.findCrewByName(crewName);
+        outputView.printAttendanceHistoryTitle(crewName);
+        final List<AttendanceResponse> attendanceResponses = convertAttendancesToResponses(crew.getAttendances());
+        outputView.printCrewAttendances(attendanceResponses);
+        final Map<AttendanceStatus, Integer> attendanceStatistics = crew.calculateAttendanceStatistics();
+        outputView.printAttendancesStatistics(attendanceStatistics);
+        final ExpulsionStatus expulsionStatus = crew.calculateExpulsionStatus();
+        outputView.printCrewExpulsionStatus(expulsionStatus);
+
+    }
```

Comment:
코드를 보니 호출 순서를 살짝 바꾸면 outputView의 인터페이스를 줄일 수 있을 것 같아 보입니다. 가능할지 확인 부탁드려요~

```suggestion
    private void responseCrewAttendanceHistory() {
        final String crewName = LoopTemplate.tryCatchLoop(this::inputCrewName, outputView);
        final Crew crew = attendanceBook.findCrewByName(crewName);
        final List<Attendance> attendances = crew.getAttendances();

        // crewName과 dto를 같이 넘겨서 outputView의 인터페이스를 줄일 수 있을 것 같습니다.
        outputView.printAttendanceHistoryTitle(crewName);
        outputView.printCrewAttendances(convertAttendancesToResponses(attendances));

        final Map<AttendanceStatus, Integer> attendanceStatistics = crew.calculateAttendanceStatistics();
        final ExpulsionStatus expulsionStatus = crew.calculateExpulsionStatus();

        // 여기 역시 attendanceStatistics와 expulsionStatus를 같이 넘겨 outputView의 인터페이스를 줄일 수 있을 것 같습니다.
        outputView.printAttendancesStatistics(attendanceStatistics);
        outputView.printCrewExpulsionStatus(expulsionStatus);

    }
```